.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
```

Comment:
I recall during collectd development that it's meant to work on python 2.6 as well, correct? If so, suggest adding a line for "2.6" so that it tests with both runtimes.